pkg/runtime-tools/generate/generate.go

```diff
@@ -103,6 +111,9 @@ func (g *Generator) Adjust(adjust *nri.ContainerAdjustment) error {
 	g.AdjustEnv(adjust.GetEnv())
 	g.AdjustHooks(adjust.GetHooks())
 	g.AdjustDevices(adjust.GetLinux().GetDevices())
+	if err := g.InjectCDIDevices(adjust.GetLinux().GetCDIDevices()); err != nil {
```

Comment:
Question: Should CDI devices be under `Linux`? They are in theory platform independent with their modifications being platform specific.